roles/network_plugin/calico/tasks/install.yml

```diff
@@ -110,10 +110,23 @@
     - calico_pool_cidr_ipv6 is defined
     - enable_dual_stack_networks
 
+- name: Calico | Remove unwanted annotations and creationTimestamp keys from metadata in Calico manifests
+  # noqa 303 - sed avoids using nested loop
+  shell: >-
+    sed -r -i
+    -e '/^\s{2,4}creationTimestamp: null$/d'
+    -e '/^\s{2,4}annotations:/{:1;/\(devel\)$/!{N;b 1}; /.*/d}'
+    {{ local_release_dir }}/calico-{{ calico_version }}-kdd-crds/*.yaml
+  when:
+    - inventory_hostname in groups['kube-master']
+    - calico_datastore == 'kdd'
+
 - name: Calico | Create calico manifests for kdd
-  template:
-    src: "{{ item.file }}.j2"
+  assemble:
+    src: "{{ local_release_dir }}/calico-{{ calico_version }}-kdd-crds"
     dest: "{{ kube_config_dir }}/{{ item.file }}"
+    regexp: ".*\\.yaml"
+    remote_src: true
   with_items:
     - {name: calico, file: kdd-crds.yml, type: kdd}
   register: calico_node_kdd_manifest
```

Comment:
This `loop` and `register` are useless (so you can also simplify the next step)
Because it's in kube_config_dir, I would name the file `calico-kdd-crd-{{ calico_version }}.yml`
I'm ok to keep the assemble step because we are going to remove the sed "soon"

Reply:
I removed the loop and the variable `calico_node_kdd_manifest` use in "Start Calico resources" but using calico version in the name would end up with leftovers after upgrades

Reply:
Not sure if there is any reasons we were applying twice the CRDs

Reply:
Could not see one, maybe a leftover